### PR TITLE
Add subcommand `wp taxonomy get`

### PR DESCRIPTION
**Pull request by rodrigoprimo**

### CODE REVIEW

**Comment by rodrigoprimo** [comment]
The first build failed because my branch was not synced with wp-cli/master. After updating it the build passed.

File: php/commands/taxonomy.php [hunk 1]
```diff
@@ -81,6 +81,59 @@ public function list_( $args, $assoc_args ) {
...
+			'name'          => $taxonomy->name,
+			'label'         => $taxonomy->label,
+			'description'   => $taxonomy->description,
+			'object_type'   => $taxonomy->object_type,
+			'show_tagcloud' => $taxonomy->show_tagcloud,
+			'hierarchical'  => $taxonomy->hierarchical,
+			'public'        => $taxonomy->public,
+			'labels'        => $taxonomy->labels,
+			'cap'           => $taxonomy->cap,
+		);
```
Comment (danielbachhuber) [inline php/commands/taxonomy.php]: Why do we need to assign this `$data` variable ?

Comment (rodrigoprimo) [inline php/commands/taxonomy.php]: @danielbachhuber `get_taxonomy()` returns fields in a seemingly random order (labels, description, public, hierarchical, show_tagcloud, cap, name, object_type and label) so I created this `$data` variable to organize them.

Comment (danielbachhuber) [inline php/commands/taxonomy.php]: Ok, fair enough.

**Pull request merged**